internal/server/pro_deputy_hub.go

```diff
@@ -1,18 +1,25 @@
 package server
 
 import (
-	"github.com/ministryofjustice/opg-sirius-supervision-pro-deputy-hub/internal/sirius"
 	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/ministryofjustice/opg-sirius-supervision-pro-deputy-hub/internal/sirius"
 )
 
 type ProDeputyHubInformation interface {
+	GetProDeputyDetails(sirius.Context, int) (sirius.ProDeputyDetails, error)
+	// GetFirmDetails(sirius.Context, int) (sirius.FirmDetails, error)
```

Comment:
There's a fair few commented out sections that I'm guessing aren't meant to be committed